assets/js/common.js

```diff
@@ -14,7 +14,7 @@ function createSearchableProductList(td, _cellData, rowData) {
   if (Array.isArray(products) && products.length > 0) {
     return `<ul>${productItems}</ul>`
   }
-  return `<ul class="no-products">No Products</ul>`
+  return `<p class="no-products">No Products</p>`
```

Comment:
No need for backticks, there's no dynamic text. They used to add processing overhead as the interpreter used to have to process the text, whereas it won't with a standard string.